roktwebviewsdk/src/main/java/rokt/com/roktwebviewsdk/RoktWebView.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ *  Copyright 2020 Rokt Pte Ltd Licensed under the Rokt Software Development Kit (SDK)
+ *  Terms of Use Version 2.0 (the "License");
+ *
+ *  You may not use this file except in compliance with the License.
+ *
+ *  You may obtain a copy of the License at https://rokt.com/sdk-license-2-0/
+ */
+
+package rokt.com.roktwebviewsdk
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.util.AttributeSet
+import android.webkit.WebView
+import rokt.com.roktwebviewsdk.Constants.ROKT_USER_AGENT_STRING
+import rokt.com.roktwebviewsdk.Constants.WEB_INTERFACE_NAME
+import java.lang.ref.WeakReference
+
+/**
+ * To use, replace instances of WebView with RoktWebView.
+ * Calls to RoktWebViewSDK.open(url) will open in an external browser.
+ */
+@SuppressLint("SetJavaScriptEnabled")
+class RoktWebView @JvmOverloads constructor(
```

Comment:
Would it be possible to have a few unittest against this class to confirm that it adds suffixes well

Reply:
Yep I'll add some UI tests